### ISSUE TITLE
Replace pseudo selector Dropdown with SsspSelectorWidget

### ISSUE THREAD

**Issue by yakutovicha** [aiidalab/aiidalab-qe#6]
Once https://github.com/aiidateam/aiida-sssp/pull/25 is merged, we should replace the Dropdown pseudo selector (with hardcoded pseudo-family names) with the `SsspSelectorWidget`

**Comment by csadorf** [comment]
For future reference, the above mentioned aiida-sssp library is superseded by https://github.com/aiidateam/aiida-pseudo/ .
